lectures/markov_chains_I.md

```diff
@@ -840,7 +854,7 @@ HTML(anim.to_jshtml())
 
 Here
 
-* $P$ is the stochastic matrix for recession and growth {ref}`considered above <mc_eg2>`.
+* $P$ is the stochastic matrix for recession and growth considered in {ref}`Example 1: Economic States <mc_eg2>`.
```

Comment:
@Jiarui-ZH can you update these links to use

https://sphinx-proof.readthedocs.io/en/latest/syntax.html#referencing-examples